wgsl/index.bs

```diff
@@ -348,6 +348,19 @@ literal_or_ident
     Specifies the binding group of the resource.
     See [[#resource-interface]].
 
+  <tr><td><dfn noexport dfn-for="attribute">`interpolate`
+    <td>One or two parameters.
+
+    The first parameter must be an [=interpolation type=].
+    The second parameter, if present, must specify the [=interpolation sampling=].
+    <td>Must only be applied to an entry point function parameter, entry point
+    return type, or member of a [=structure=] type.
+    Must only be applied to declarations of scalars or vectors of floating-point type.
+    Must not be used with the [=compute=] shader stage.
```

Comment:
nit:  Can we be more constructivist here:

> Must only be used with a vertex or fragment shader stage.

Reply:
Should this also say:

> The attribute is significant only when used on a vertex output or a fragment input.

But I see the potential conflict to make it an error for compute shaders, and "only signifcant" in other cases.

So maybe:

> Must only be used with a vertex or fragment shader stage.
> For a vertex entry point, the attribute is significant only on outputs.
> For a fragment entry point, the attribute is significant only on inputs.

Reply:
I generally phrase it this way because it seems easier to expand in the future to me. Location is similarly described as not compute. Should this change be applied generally?

Reply:
I'm ok to keep the "not compute" wording.  It's only a small hazard to future growth.

My main concern is that interpolation is only significant on vertex outputs and fragment inputs;  if that's true, then the spec should say it.